ibft/eventqueue/queue.go

```diff
@@ -1,36 +1,54 @@
 package eventqueue
 
-import "sync"
+import (
+	"github.com/bloxapp/ssv/utils/logex"
+	"go.uber.org/zap"
+	"sync"
+	"time"
+)
 
-// Queue is a simple struct that manages a queue of functions. Thread safe
-type Queue struct {
+// Event represent some function
+type Event func()
+
+// EventQueue is the interface for managing a queue of functions
+type EventQueue interface {
```

Comment:
@nivBlox why do we need Ann interface? Even if we do it should be in a different PR

Reply:
why in differ pr?